src/main/resources/default/templates/biz/process/process-logs.html.pasta

```diff
@@ -1,13 +1,27 @@
 <i:arg type="sirius.biz.process.Process" name="process"/>
 <i:arg type="Page" name="logs"/>
+<i:arg type="boolean" name="isAutoRefreshEnabled"/>
 
 <i:invoke template="/templates/biz/process/process.html.pasta"
           process="process"
-          page="logs">
+          page="logs"
+          isAutoRefreshEnabled="isAutoRefreshEnabled">
     <i:block name="sidebar">
         <t:filterbox page="logs" baseUrl="@apply('/ps/%s', process.getId())"/>
     </i:block>
     <i:block name="header-actions">
+        <i:if test="process.shouldAutorefresh()">
+            <a id="enableAutoRefresh" class="btn btn-link d-none" href="@apply('/ps/%s?isAutoRefreshEnabled=true', process.getId())">@i18n("Process.enableAutoRefresh")</a>
+            <a id="disableAutoRefresh" class="btn btn-link d-none" href="@apply('/ps/%s?isAutoRefreshEnabled=false', process.getId())">@i18n("Process.disableAutoRefresh")</a>
```

Comment:
Check via i:if directly which button should be rendered instead of JS?

Reply:
Yeah, actualy that might be a good idea. I am going to check it out. =)

Reply:
Done